### PR TITLE
Bring product features in quickview when combinations exist

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Bring the feature available on the product page when the customer select another combination (Page update ...) |
| Type? | bug fix |
| Category? | FO |
| BC breaks? | Nope |
| Deprecations? | Nope |
| Fixed ticket? | <ul><li>[BOOM-361](http://forge.prestashop.com/browse/BOOM-361)</li><li>[BOOM-836](http://forge.prestashop.com/browse/BOOM-836)</li></ul> |
| How to test? | On a product quickview, you should not have to click 10 times before another color is selected. Choosing another combination must update the quickview content (Images, availability, price etc.) |

### CODE REVIEW

**Comment by vincentbz** [comment]
Tested and ok, thanks !
Tested with images, prices, availability, combinations...

**Comment by mickaelandrieu** [comment]
thanks @Quetzacoalt91

File: themes/classic/templates/catalog/_partials/quickview.tpl [hunk 1]
```diff
@@ -30,13 +30,18 @@
...
                 {block name='product_variants'}
                   {include file='catalog/_partials/product-variants.tpl'}
                 {/block}
 
                 {block name='product_add_to_cart'}
                   {include file='catalog/_partials/product-add-to-cart.tpl'}
                 {/block}
+                
+                {block name='product_refresh'}
+                  <input class="product-refresh" data-url-update="false" name="refresh" type="submit" value="{l s='Refresh' d='Shop.Theme.Actions'}" hidden>
```
Comment (maximebiloe) [inline themes/classic/templates/catalog/_partials/quickview.tpl]: Is the `hidden` as an attribute wanted?

Comment (Quetzacoalt91) [inline themes/classic/templates/catalog/_partials/quickview.tpl]: Yes, the JS file checks the existence of this input in the DOM in order to launch the auto refresh, but the customer won't need to interact with it directly.

**Pull request merged**